charts/n8n/templates/pvc.yaml

```diff
@@ -2,7 +2,7 @@
 apiVersion: v1
```

Comment:
If `.Values.main.persistence.enabled` is set to false, shouldn't we have an if statement here as well to not create the PVC?